### PR TITLE
Fixed 2 warnings by removing a redundant access modifier

### PR DESCRIPTION
**Pull request by ghost**
Fixed 2 warnings by removing the redundant `public` access modifier from both `ynab` and `yyyyMMdd` properties in DateFormatter extension.

### CODE REVIEW

**Comment by ghost** [comment]
@dauby88 Would be lovely if you can take a look when you have a chance

**Comment by dauby88** [comment]
Looks good! Thanks!

**Review by dauby88 — APPROVED** [review]

**Comment by ghost** [comment]
Apparently I can't merge it even if you approve it, you have to do it yourself since I don't have "Write access" to the repo.

**Pull request merged**
